packages/dev/core/src/FlowGraph/Blocks/Data/flowGraphConstantBlock.ts

```diff
@@ -0,0 +1,25 @@
+import { FlowGraphBlock } from "core/FlowGraph/flowGraphBlock";
+import type { FlowGraphContext } from "core/FlowGraph/flowGraphContext";
+import type { FlowGraphDataConnection } from "core/FlowGraph/flowGraphDataConnection";
+import { getRichTypeFromValue } from "core/FlowGraph/flowGraphRichTypes";
+
+export interface IFlowGraphConstantBlockConfiguration<T> {
+    value: T;
+}
+/**
+ * @experimental
+ * Block that returns a constant value.
+ */
+export class FlowGraphConstantBlock<T> extends FlowGraphBlock {

```

Comment:
wondering what is the difference with a value block ? and how in practice (thinking editor here) how differently it would behave ?

Reply:
No difference, it's just how I'm calling the Value block but I can rename if it's clearer

Reply:
Ohhh I thought you had one already :-) so all good for me.